help/destinations/catalog/mobile-engagement/line.md

```diff
@@ -0,0 +1,141 @@
+
+Fill in the required fields below.
+* **[!UICONTROL Bearer token]**: Your [[!DNL LINE Channel access token (long-lived)] from the LINE developer console](#gather-credentials).
+
+If the details provided are valid, the UI displays a **Connected** status with a green check mark. You can then proceed to the next step.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+![Platform UI screenshot showing the destination details.](../../assets/catalog/mobile-engagement/line/destination-details.png)
+
+* **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+* **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+* **[!UICONTROL Audience Type]**: If the identities in Platform are of type *IDFA* select **[!UICONTROL IDFA]**. If the identities in Platform are of type *UserID* select **[!UICONTROL UserID]**. Refer to [Supported identities](#supported-identities) if you need guidance.
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+> To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map attributes and identities {#map}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL LINE] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the [!DNL LINE] destination fields, follow these steps:
+
+The following target identity namespace(s) must be mapped:
+* **UserID** must be mapped as a target field using **Target field** > **[!UICONTROL Select identity namespace]** > **LineId**
```

Comment:
I am confused by this section. We are saying that "**UserID** must be mapped as a target field". 

I have a few comments: 

1. Did we actually mean as a source field, since it looks to be on the left side of the screenshot, as source field?
2. Earlier in the doc we were also talking about IDFA identities. Can we document what happens if the identities we are looking to export are of the type IDFA?

Reply:
1. Content changed.
2. Updated documentation.